test/commands/promote.js

```diff
@@ -54,15 +59,13 @@ describe('pg:promote when argument is database', () => {
       confirm: 'myapp'
     }).reply(201)
     return cmd.run({app: 'myapp', args: {}, flags: {}})
-    .then(() => expect(cli.stderr, 'to equal', `Ensuring an alternate alias for existing DATABASE_URL... RED_URL
-Promoting postgres-1 to DATABASE_URL on myapp... done
-`))
+    .then(() => expect(cli.stderr, 'to equal', expectedMessage.replace('PURPLE', 'postgres-1')))
```

Comment:
I like your thinking here, but honestly I find it easier to work with repeated strings, and copy and paste them from failing test output as necessary (since there it's easy to compare). I don't feel strongly about this, though.

Reply:
makes sense. updated